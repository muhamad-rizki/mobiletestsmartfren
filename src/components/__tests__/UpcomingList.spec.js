/* eslint-disable no-undef */
import React from 'react';
import { shallow } from 'enzyme';

import {
  UpcomingList,
} from '../index';

describe('UpcomingList Component', () => {
  it('renders as expected', () => {
    const setUCLoading = jest.fn();
    const wrapper = shallow(
      <UpcomingList
        upcoming={{
          results: [],
        }}
        imgUrl="https://image.tmdb.org/t/p/"
        upcomingIndex={0}
        setUCLoading={setUCLoading}
        loadingUpcoming={false}
        activeTabIndex={1}
        // eslint-disable-next-line jsx-a11y/tabindex-no-positive
        tabIndex={1}
      />,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
