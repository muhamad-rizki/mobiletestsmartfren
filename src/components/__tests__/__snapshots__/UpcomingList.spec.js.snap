// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UpcomingList Component renders as expected 1`] = `ShallowWrapper {}`;
